cmd/vulcan-docker-image/vulcan-docker-image.go

```diff
@@ -103,7 +103,7 @@ func main() {
 	c.RunAndServe()
 }
 
-func run(ctx context.Context, target string, optJSON string, state state.State) error {
+func run(ctx context.Context, target, assetType, optJSON string, state state.State) error {
```

Comment:
Are we skipping the IsReachable on purpose?

Reply:
This one I skipped because I think is deprecated in favor of Trivy, right?

Reply:
That's why I was asking. 👍